Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -130,13 +153,9 @@
 
     m_parsedManifest = true;
 
-    // FIXME: This needs to check the bundle code signature (if a bundle is used).
-    NSURL *manifestURL = [NSURL fileURLWithPath:@"manifest.json" relativeToURL:m_resourceBaseURL.get()];
-    NSData *manifestData = [NSData dataWithContentsOfURL:manifestURL];
-    if (!manifestData) {
-        recordError(createError(Error::ManifestNotFound));
+    NSData *manifestData = resourceDataForPath(@"manifest.json");
+    if (!manifestData)
         return nil;
```

Comment:
Do we want any of the errors here? Or are these handled elsewhere?

Reply:
It is handled by the more generic ResourceNotFound error and the resourceData() method.